ocs_ci/ocs/ocp.py

```diff
@@ -840,6 +840,43 @@ def get_logs(
         )
         return output
 
+    def get_build(self):
+        """
+        Return the OCP Build Version
+
+        Args:
+            None
+
+        Returns:
+             str: The build version of the OCP
```

Comment:
add an empty line below to comply with Google sphinx style - https://www.sphinx-doc.org/en/1.5/ext/example_google.html

Please fix in all plcaes

Reply:
update in all places